ext/notes/main.php

```diff
@@ -244,7 +244,7 @@ private function add_new_note(): int
 
         Log::info("notes", "Note added {$noteID} by " . Ctx::$user->name);
 
-        $database->execute("UPDATE images SET notes=(SELECT COUNT(*) FROM notes WHERE image_id=:id) WHERE id=:id", ['id' => $note['image_id']]);
+        Ctx::$database->execute("UPDATE images SET notes=(SELECT COUNT(*) FROM notes WHERE image_id=:id AND enable=1) WHERE id=:id", ['id' => $note['image_id'], 'enable' => 1]);
```

Comment:
This has `enable=1` hard-coded in the SQL query _and_ a parameter of `enable => 1` :eyes: